import "libnutrition/proto/data/user.proto";
import "libnutrition/proto/service/error.proto";

message UserLoadRequest
{
	optional bool omit = 1 [default = false];
	optional bool all = 2 [default = false];
	repeated int32 requestedIds = 3;
	repeated string requestedUsernames = 4;
}

message UserLoadResponse
{
	repeated UserData users = 1;
	optional Error error = 100;
}

message UserStoreRequest
{
	repeated UserData users = 1;
}

message UserStoreResponse
{
	repeated int32 storedIds = 1;
	optional Error error = 100;
}

